openms-wiki/build-openms-from-source.md

```diff
@@ -0,0 +1,10 @@
+Build OpenMS From Source
+========================
```

Comment:
The directory structure is incorrect:

Add this file inside: `OpenMS-docs/docs/installations`.

Reply:
done